tee-worker/litentry/core/credentials/src/litentry_profile/token_balance.rs

```diff
@@ -81,27 +106,68 @@ fn update_assertion(token: ETokenAddress, balance: f64, credential: &mut Credent
 
 fn get_token_info(token: &ETokenAddress) -> (&'static str, &'static str) {
 	match token {
-		ETokenAddress::Lit | ETokenAddress::Usdc | ETokenAddress::Usdt | ETokenAddress::Wbtc =>
-			VC_TOKEN_BALANCE_INFOS[0],
+		ETokenAddress::WBTC
+		| ETokenAddress::LIT
+		| ETokenAddress::USDC
+		| ETokenAddress::USDT
+		| ETokenAddress::CRV
+		| ETokenAddress::MATIC
+		| ETokenAddress::DYDX
+		| ETokenAddress::AMP
+		| ETokenAddress::CVX
+		| ETokenAddress::TUSD
+		| ETokenAddress::USDD
+		| ETokenAddress::GUSD
+		| ETokenAddress::LINK
+		| ETokenAddress::GRT
+		| ETokenAddress::COMP
+		| ETokenAddress::PEOPLE
+		| ETokenAddress::GTC => VC_TOKEN_BALANCE_INFOS[0],
 		_ => ("UnknownType", ("UnkonwDescription")),
 	}
 }
 
 fn get_assertion_content(token: &ETokenAddress) -> &'static str {
 	match token {
-		ETokenAddress::Lit => "$lit_holding_amount",
-		ETokenAddress::Usdc => "$usdc_holding_amount",
-		ETokenAddress::Usdt => "$usdt_holding_amount",
-		ETokenAddress::Wbtc => "$wbtc_holding_amount",
+		ETokenAddress::WBTC => "$wbtc_holding_amount",
+		ETokenAddress::LIT => "$lit_holding_amount",
+		ETokenAddress::USDC => "$usdc_holding_amount",
+		ETokenAddress::USDT => "$usdt_holding_amount",
+		ETokenAddress::CRV => "$lit_holding_amount",
+		ETokenAddress::MATIC => "$usdc_holding_amount",
+		ETokenAddress::DYDX => "$usdt_holding_amount",
+		ETokenAddress::AMP => "$wbtc_holding_amount",
+		ETokenAddress::CVX => "$lit_holding_amount",
+		ETokenAddress::TUSD => "$usdc_holding_amount",
+		ETokenAddress::USDD => "$usdt_holding_amount",
+		ETokenAddress::GUSD => "$wbtc_holding_amount",
+		ETokenAddress::LINK => "$lit_holding_amount",
+		ETokenAddress::GRT => "$usdc_holding_amount",
+		ETokenAddress::COMP => "$usdt_holding_amount",
+		ETokenAddress::PEOPLE => "$wbtc_holding_amount",
+		ETokenAddress::GTC => "$wbtc_holding_amount",
```

Comment:
Since this VC is not live, could we remove $wbtc_ from the "$wbtc_holding_amount", and add it in a seperate assertion condition, e.g.: 

{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://w3id.org/security/suites/ed25519-2020/v1"
  ],
  "id": "0x11cea83e5f816ca48b0937c514779c2f0629a9039347ece216dc850635aee3ee",
  "type": [
    "VerifiableCredential"
  ],
  "credentialSubject": {
    "id": "did:litentry:substrate:0xd4e35b16ec6b417386b948e7eaf5cc642a243096cecf366e6313689b90969f42",
    "description": "The amount of a particular token you are holding",
    "type": "Token Holding Amount",
    "assertions": [
      {
        "and": [
          {
            "src": "$token",
            "op": "==",
            "dst": "TON"
          },
          {
            "or": [
              {
                "and": [
                  {
                    "src": "$network",
                    "op": "==",
                    "dst": "Ethereum"
                  },
                  {
                    "src": "$address",
                    "op": "==",
                    "dst": "0x582d872a1b094fc48f5de31d3b73f2d9be47def1"
                  }
                ]
              },
              {
                "and": [
                  {
                    "src": "$network",
                    "op": "==",
                    "dst": "Bsc"
                  },
                  {
                    "src": "$address",
                    "op": "==",
                    "dst": "0x76a797a59ba2c17726896976b7b3747bfd1d220f"
                  }
                ]
              }
            ]
          },
          {
            "src": "$holding_amount",
            "op": ">=",
            "dst": "3000"
          }
        ]
      }
    ],
    "values": [
      true
    ],
    "endpoint": "http://localhost:9933/"
  },
  "issuer": {
    "id": "did:litentry:substrate:0x3f27438fd2f7841d8296eb5cf62936c8675b86d631c530db2a8036ee064f7354",
    "name": "Litentry TEE Worker",
    "mrenclave": "DPKnrdXcuy7Mewm74vagKpstQctwUFokzwLN5JMkabRu"
  },
  "issuanceDate": "2023-12-14T01:21:30.880890053+00:00",
  "proof": {
    "created": "2023-12-14T01:21:30.881938825+00:00",
    "type": "Ed25519Signature2020",
    "proofPurpose": "assertionMethod",
    "proofValue": "a574ae32e1f33939f125a5b979b4dd6b7d29e15d6549ffbdd946a926769898887c1afe4af94581f451c5a1701139475e6dc971f9999823f247f3a8e3cf68b90a",
    "verificationMethod": "0x3f27438fd2f7841d8296eb5cf62936c8675b86d631c530db2a8036ee064f7354"
  }
}

Reply:
I think it's because time is tight, and the previous method can be completed relatively quickly.